IonicPortals/IonicPortals/PortalManager.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import UIKit
 
 /// Maintains a registry of ``Portal``s for the lifecycle of an application
-@objc(PortalManager)
+@objc(IONPortalManager)
```

Comment:
I decided to follow conventional Objective-C naming conventions for our `@objc` classes